common/script/ops/clearPMs.js

```diff
@@ -1,6 +1,6 @@
 module.exports = function clearPMs (user) {
   user.inbox.messages = {};
-  user.markModified('inbox.messages');
+  // user.markModified('inbox.messages');
```

Comment:
for that case I added https://github.com/HabitRPG/habitrpg/blob/api-v3/common/script/index.js#L236-L239